src/sqlast/mod.rs

```diff
@@ -608,3 +614,28 @@ impl FromStr for FileFormat {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct SQLDrop {
+    pub if_exists: bool,
+    pub names: Vec<SQLObjectName>,
+    pub cascade: bool,
+    pub restrict: bool,
```

Comment:
Like with `SELECT ALL` (#76) I wonder if it makes sense to store the `restrict` in the AST, given it's the default.

Reply:
Agreed, fixed.